vowel_duplicates.py

```diff
@@ -2,17 +2,17 @@
 def vowels(word):
     new_word = []
     vowel = ['a', 'e', 'i', 'o', 'u']
-    for x in word:
-        if x in vowel:
-           new_word.append(x)
-           count = new_word.count(x)
-           if count > 1:
-               new_word.pop(1)
-               count = count-1
-               return new_word, count
+    #get number of duplicates using set function
+    duplicates = len(word) - len(set(word))
 
-    return new_word
+    for x in set(word):
```

Comment:
You can even use sets throughout instead of looping i.e create a set of vowels and a set from the word, There intersection will be the present vowels

Reply:
Thanks @dennisja , I have tried it out in these new changes